collector/test/ProcessSignalFormatterTest.cpp

```diff
@@ -607,8 +610,8 @@ TEST(ProcessSignalFormatterTest, Rox3377ProcessLineageWithNoVPidTest) {
 
   EXPECT_EQ(lineage.size(), 1);
 
-  EXPECT_EQ(lineage[0].parent_uid(), tinfo2->m_user.uid);
-  EXPECT_EQ(lineage[0].parent_exec_file_path(), tinfo2->m_exepath);
+  EXPECT_EQ(lineage[0].parent_uid(), 7);
+  EXPECT_EQ(lineage[0].parent_exec_file_path(), "asdf");
```

Comment:
Interesting, why replacing this and other `EXPECT_EQ` examples with a static value, instead of getting it from `tinfo`?

Reply:
This is the most unfortunate fallout from Falco's "ownership" changes. Once we `std::move` the `unique_ptr` into the inspector, it's no longer valid and is essentially NULL. So once we've populated the inspector with threads we have essentially lost the information. 

I fiddled with this _a lot_ but because Falco explicitly deletes copy and default constructors there seems to be no way we can conform to the `unique_ptr` requirement without also having to hardcode expected values like this.